packages/graphql/lib/mock-server-middleware.js

```diff
@@ -4,31 +4,31 @@ import express from 'express';
 import hopsConfig from 'hops-config';
 import schema from 'hops-graphql/schema';
 
-const app = express();
-app.use(cookieParser());
-
-const schemaPromise = Promise.resolve(
+const apolloAppPromise = Promise.resolve(
   typeof schema === 'function' ? schema() : schema
-);
-
-export default (req, res, next) => {
-  schemaPromise.then(schema => {
-    const server = new ApolloServer({
-      schema,
-      context: ({ req }) => ({ req, config: hopsConfig }),
-    });
+).then(resolvedSchema => {
+  const app = express();
+  app.use(cookieParser());
 
-    server.applyMiddleware({
-      app,
-      path: hopsConfig.graphqlMockServerPath,
-      cors: {
-        credentials: true,
-        origin: true,
-        methods: 'GET,HEAD,PUT,PATCH,POST,DELETE',
-        allowedHeaders: 'content-type',
-      },
-    });
+  const server = new ApolloServer({
+    schema: resolvedSchema,
+    context: context => ({ ...context, config: hopsConfig }),
```

Comment:
So this now exposes at least the `Response` object to the resolvers. Do we really want this?

Reply:
That was the idea but reading the documentation it seems that we only have the req object there.

> When context is defined as a function, it will be called on each request and receive an object containing a req property which represents the request.